editor/project_manager/project_dialog.cpp

```diff
@@ -315,6 +315,8 @@ void ProjectDialog::_create_dir_toggled(bool p_pressed) {
 			target_path = target_path.path_join(last_custom_target_dir);
 		}
 	} else {
+		// Strip any trailing slash.
+		target_path = target_path.rstrip("/").rstrip("\\")
```

Comment:
Isn't a semi-colon `;` needed at the end of the line?
```suggestion
		target_path = target_path.rstrip("/").rstrip("\\");
```

Reply:
Absolutely 😅 I can't believe I missed that